mockito-core/src/main/java/module-info.java

```diff
@@ -30,6 +30,8 @@
     exports org.mockito.session;
     exports org.mockito.stubbing;
     exports org.mockito.verification;
+    exports org.mockito.internal to
+            java.instrument;
```

Comment:
I know it might sound annoying, but is it possible to add a regression test for this? Given these things are so subtle, I do prefer to have tests written so I can confidently merge other PRs knowing I don't inadvertently break this setup. If it isn't possible, then alas.

Reply:
This should already fail without, but I think recent VMs fix this export implicitly.